segmentation/utils/spatial_analysis.py

```diff
@@ -56,199 +55,201 @@ def calculate_channel_spatial_enrichment(dist_matrices, marker_thresholds, all_d
         Args:
-            all_data: data for the all fovs in the form of a pandas DF, including the columns of SampleID (fovs),
+            all_data (pandas df): data for the all fovs in the form of a pandas DF, including the columns of SampleID (fovs),
                 cellLabelInImage (the cell label), and FlowSOM_ID (the cell phenotype id).
-            dist_matrices: A dictionary that contains a cells x cells matrix with the euclidian
+            dist_matrices_dict (dict): A dictionary that contains a cells x cells matrix with the euclidian
                 distance between centers of corresponding cells for every fov
-            fov_list: patient labels to include in analysis. If argument is none, default is all labels used.
-            distlim: cell proximity threshold. Default is 50.
-            fov_col: column with the cell fovs (Default is SampleID)
-            flowsom_col: column with the cell phenotype IDs (Default is FlowSOM_ID)
-            cell_label_col: column with the cell labels (Default is cellLabelInImage)
-            cell_type_col: column with the cell types (Default is cell_type)
+            included_fovs (list): patient labels to include in analysis. If argument is none, default is all labels used.
+            distlim (int): cell proximity threshold. Default is 50.
+            fov_col (string): column with the cell fovs (Default is SampleID)
+            cluster_id_col (string): column with the cell phenotype IDs (Default is FlowSOM_ID)
+            cell_label_col (string): column with the cell labels (Default is cellLabelInImage)
+            cluster_name_col (string): column with the cell types (Default is cell_type)
         Returns:
-            cell_neighbor_counts: matrix with phenotype counts per cell
-            cell_neighbor_freqs: matrix with phenotype frequencies of
+            cell_neighbor_counts (pandas df): matrix with phenotype counts per cell
+            cell_neighbor_freqs (pandas df): matrix with phenotype frequencies of
                 counts per phenotype/total phenotypes for each cell"""
 
     # Setup input and parameters
-    if fov_list is None:
-        fov_list = sorted(list(set(all_data[fov_col])))
+    if included_fovs is None:
+        included_fovs = sorted(list(set(all_data[fov_col])))
 
     # Error Checking
-    if not np.isin(fov_list, all_data[fov_col]).all():
+    if not np.isin(included_fovs, all_data[fov_col]).all():
         raise ValueError("Points were not found in Expression Matrix")
 
     # Get the phenotypes
-    pheno_titles = all_data[cell_type_col].drop_duplicates()
+    cluster_names = all_data[cluster_name_col].drop_duplicates()
 
-    # Subset just the sampleID, cellLabelInImage, and FlowSOMID
-    all_data = all_data[[fov_col, cell_label_col, flowsom_col, cell_type_col]]
+    # Subset just the sampleID, cellLabelInImage, and FlowSOMID, and cell phenotype
+    all_fov_neighborhood_data = all_data[[fov_col, cell_label_col, cluster_id_col, cluster_name_col]]
```

Comment:
change this to be all_neighborhood_data to be consistent with the naming pattern of the two functions above